README.md

```diff
@@ -48,13 +48,13 @@ go run build/make.go --all-platforms
 After compilation
 
 ```
-go run build/make.go --install
+go run build/make.go install
 ```
 
 Installing to a CUSTOM_LOCATION
 
 ```
-go run build/make.go --install --plugin-prefix CUSTOM_LOCATION
```

Comment:
This can remain 
`go run build/make.go --install --plugin-prefix CUSTOM_LOCATION`